### PR TITLE
Add sitemap

### PR DESCRIPTION
**Pull request by carlbarrdahl**

### CODE REVIEW

**Comment by owocki** [comment]
once this is merged i can ingest these docs into gitcoin.ai!

**Review by codenamejason — APPROVED** [review]
Looking forward to seeing what comes out of this with the AI 🔥

**Comment by owocki** [comment]
anything blocking merging?  i wanna play with this in gitcoin.ai

**Pull request merged**
